warehouse/query-core/src/main/java/datawave/query/index/lookup/RangeStream.java

```diff
@@ -442,112 +442,109 @@ public ScannerStream visit(ASTEQNode node, Object data) {
-                    return ScannerStream.unindexed(node);
+                final IteratorSetting uidSetting = new IteratorSetting(stackStart++, iterClazz);
+                
+                if (setCondenseUids) {
+                    uidSetting.addOption(CondensedUidIterator.SHARDS_TO_EVALUATE, Integer.valueOf(config.getShardsPerDayThreshold()).toString());
+                    uidSetting.addOption(CondensedUidIterator.MAX_IDS, Integer.valueOf(MAX_MEDIAN).toString());
+                    uidSetting.addOption(CondensedUidIterator.COMPRESS_MAPPING, Boolean.valueOf(compressUidsInRangeStream).toString());
+                    
+                    if (condensedTld) {
+                        uidSetting.addOption(CondensedUidIterator.IS_TLD, Boolean.valueOf(condensedTld).toString());
+                    }
                 }
-            } catch (TableNotFoundException e) {
-                log.error(e);
-                throw new RuntimeException(e);
+                uidSetting.addOption(CreateUidsIterator.COLLAPSE_UIDS, Boolean.valueOf(config.getCollapseUids()).toString());
+                
+                options.addScanIterator(uidSetting);
+                StringBuilder queryString = new StringBuilder(fieldName);
+                queryString.append("=='").append(literal).append("'");
+                options.addScanIterator(QueryScannerHelper.getQueryInfoIterator(config.getQuery(), false, queryString.toString()));
+                
+                scanSession.setRanges(Collections.singleton(rangeForTerm(literal, fieldName, config))).setOptions(options);
+                
+                itr = Iterators.transform(scanSession, new EntryParser(node, fieldName, literal, indexOnlyFields));
+                
+            } else {
+                
+                BatchScanner scanner = scanners.newScanner(config.getIndexTableName(), config.getAuthorizations(), 1, config.getQuery());
+                
+                scanner.setRanges(Collections.singleton(rangeForTerm(literal, fieldName, config)));
+                scanner.fetchColumnFamily(new Text(fieldName));
+                scanner.addScanIterator(makeDataTypeFilter(config, stackStart++));
+                
+                final IteratorSetting uidSetting = new IteratorSetting(stackStart++, createUidsIteratorClass);
+                uidSetting.addOption(CreateUidsIterator.COLLAPSE_UIDS, Boolean.valueOf(config.getCollapseUids()).toString());
+                scanner.addScanIterator(uidSetting);
+                
+                itr = Iterators.transform(scanner.iterator(), new EntryParser(node, fieldName, literal, indexOnlyFields));
             }
-            log.debug("{\"" + fieldName + "\": \"" + literal.toString() + "\"} is not an observed field.");
-            return ScannerStream.unknownField(node);
+            
+            /**
+             * Create a scanner in the initialized state so that we can
```

Comment:
so that we can...

I know you didn't change this, but i feel like we should complete the sentence or delete this comment

Reply:
Just remove the comment....it is useless